track/templates/includes/en/header.html

```diff
@@ -2,6 +2,8 @@
 	<h1 class="text-4xl sm:text-5xl">Check compliance</h1>
 	<h2 class="mb-4 sm:mb-6 text-xl sm:text-2xl">Last updated {{ scan_date | display_date("en") }}</h2>
 	<p class="text-lg mb-6">This dashboard reports how federal government websites and web services are meeting good web security practices, as outlined in <a class="text-https-blue hover:text-black font-bold" href="https://www.canada.ca/en/treasury-board-secretariat/services/information-technology/policy-implementation-notices/implementing-https-secure-web-connections-itpin.html">Information Technology Policy Implementation Notice (ITPIN): Implementing HTTPS for Secure Web Connections</a>.</p>
-	<p class="text-lg mb-8"><a id="modal-btn" class="font-bold text-https-blue hover:text-black" href="#">How do I read this table?</a></p>
+	<p class="text-lg mb-8"><a id="modal-btn" class="font-bold text-https-blue hover:text-black" href="#">How do I read this table?</a> <a href="https://canada-ca.github.io/pages/submit-institutional-domains.html" class="font-bold text-https-blue hover:text-black"> Submit your institution's domains</a></p>
```

Comment:
Do you think it's really necessary to put the submit link both here and in the nav bar?

It looks kinda janky to me.

 
![screen shot 2018-07-25 at 11 05 52 am](https://user-images.githubusercontent.com/1952571/43209508-bf9fbb98-8ffa-11e8-99a3-ec210a4cc099.png)

Reply:
We decided to put it there also for when Menu only appeared  for smaller screen .... maybe we can add a class to show it only for smaller screen.